server/app.ts

```diff
@@ -41,7 +41,7 @@ export default function createApp(services: Services): express.Application {
   app.use(setUpWebRequestParsing())
   app.use(setUpStaticResources())
   app.use(setUpAuthentication())
-  app.use(authorisationMiddleware())
+  app.use(authorisationMiddleware(['ROLE_ACTIVITY_HUB']))
   app.use(setUpSuccessMessages())
```

Comment:
Just checking this - the service should be accessible by anyone regardless of role, but some tiles and features will be configured to require the ROLE_ACTIVITY_HUB. Does this change REQUIRE ROLE_ACTIVITY_HUB to even enter the service? If so, I'm afraid that's not the intention.